README.md

```diff
@@ -1,2 +1,145 @@
 # redux-needs &middot; [![Build Status](https://travis-ci.org/LUKKIEN/redux-needs.svg?branch=master)](https://travis-ci.org/LUKKIEN/redux-needs) [![codecov](https://codecov.io/gh/LUKKIEN/redux-needs/branch/master/graph/badge.svg)](https://codecov.io/gh/LUKKIEN/redux-needs)
 
+> 🎁 Wrapping everyday data to your containers in need 🎁
+
+Bind actions to changes in your redux state based on the needs of your active
+redux components.
+
+[Usage](#usage) |
```

Comment:
Maybe we can add a chapter on compatibility / requirements?
- react: 15.5.x
- react-redux: 5.x.x
- redux: 3.7.x